pkg/components/enterprise.go

```diff
@@ -157,6 +157,11 @@ var (
 		Image:   "tigera/packetcapture",
 	}
 
+	ComponentPolicyRecommendation = component{
```

Comment:
will need an entry in hack/gen-versions/enterprise.go.tpl instead - that file generates this one iirc